testutils/helper/install.go

```diff
@@ -93,7 +93,12 @@ func NewSoloTestHelper(configFunc TestConfigFunc) (*SoloTestHelper, error) {
 		}
 		testConfig.version = version
 	} else {
-		testConfig.version = testConfig.ReleasedVersion
+		// we use the version field as a chart version and tests assume it doesn't have a leading 'v'
```

Comment:
can you add a comment in the TestConfig struct above, with the expected format of the `version` and `ReleasedVersion` fields (i.e. one cannot have a leading `v` and one may have it but it will be stripped)